components/text-utilities/text-utilities.html

```diff
@@ -0,0 +1,179 @@
+                <h2 class="text_left">text-align: left</h2>
+                <h2 class="text_center">text-align: center</h2>
+                <h2 class="text_right">text-align: right</h2>
+            </div>
+
+            <div class="display_code">
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=material&wt=none&l=css&width=680&ds=true&dsyoff=0px&dsblur=0px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Droid+Sans+Mono&fs=14.5px&lh=147%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522text_box%2522%253E%250A%2509%253Ch2%2520class%253D%2522text_left%2522%253Etext-align%253A%2520left%253C%252Fh2%253E%250A%2509%253Ch2%2520class%253D%2522text_center%2522%253Etext-align%253A%2520center%253C%252Fh2%253E%250A%2509%253Ch2%2520class%253D%2522text_right%2522%253Etext-align%253A%2520right%253C%252Fh2%253E%250A%253C%252Fdiv%253E"
+                    style="width: 482px; height: 181px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+
+        </div> 
+
+        <!-- Text Sizes -->
+
+        <div class="doc_main_container">
+            <h2 class="doc_main_head">Text Sizes</h2>
+            <p class="doc_main_desc">
+                Text Alignment is use to align the text on web page. Right, Left, And Center. Here, You can use these classes to align text text_left,
+                 text_center and text_right.
+            </p>
+
+            <div class="text_box">
+                <p class="paragraph text_xl">Extra large Text Size</p>
+                <p class="paragraph text_lg">large Text Size</p>
+                <p class="paragraph text_md">Medium Text Size</p>
+                <p class="paragraph text_sm">Small Text Size</p>
+                <p class="paragraph text_xm">Extra small Text Size</p>
+            </div>
+
+            <div class="display_code">
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=material&wt=none&l=css&width=680&ds=true&dsyoff=0px&dsblur=0px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Droid+Sans+Mono&fs=14.5px&lh=147%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522text_box%2522%253E%250A%2509%253Cp%2520class%253D%2522paragraph%2520text_xl%2522%253EExtra%2520large%2520Text%2520Size%253C%252Fp%253E%250A%2509%253Cp%2520class%253D%2522paragraph%2520text_lg%2522%253Elarge%2520Text%2520Size%253C%252Fp%253E%250A%2509%253Cp%2520class%253D%2522paragraph%2520text_md%2522%253EMedium%2520Text%2520Size%253C%252Fp%253E%250A%2509%253Cp%2520class%253D%2522paragraph%2520text_sm%2522%253ESmall%2520Text%2520Size%253C%252Fp%253E%250A%2509%253Cp%2520class%253D%2522paragraph%2520text_xm%2522%253EExtra%2520small%2520Text%2520Size%253C%252Fp%253E%250A%253C%252Fdiv%253E"
+                    style="width: 542px; height: 223px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
you can increase width and height a bit so that the vertical scroll bar at the right of the carbon container will not display on UI.